coding_style/README.adoc

```diff
@@ -150,6 +150,16 @@ Rationale:: https://yaml.org/type/bool.html[YAML 1.1] allows all variants wherea
   But do use quotes for user-side strings such as descriptions, names, and messages.
 * Even if JSON is valid YAML and Ansible understands it, do only use JSON syntax if it makes sense (e.g. a variable file automatically generated) or adds to the readability.
   In doubt, nobody expects JSON so stick to YAML.
+* A single jinja template should not exceed 250 lines in length
+* The aggregate number of jinja includes should not exceed 3
```

Comment:
What do you call aggregate number?

Reply:
Yeah this combined with the 250 line limit above seems arbitrary and potentially a pain.